tendrl/commons/objects/cluster_node_context/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import socket
+import uuid
+
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.commons.utils import cmd_utils
+
+from tendrl.commons import objects
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ClusterNodeContext(objects.BaseObject):
+
+    def __init__(self, machine_id=None, node_id=None, fqdn=None,
+                 tags=None, status=None, *args, **kwargs):
+        super(ClusterNodeContext, self).__init__(*args, **kwargs)
+        self.value = 'clusters/%s/nodes/%s/NodeContext'
+        self.machine_id = machine_id or NS.node_context.machine_id
```

Comment:
Do a "node_context = NS.node_context.load()"

the use "node_context.machine_id" and other attributes